index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Please add a descriptive README file to your project, to tell other people why your project is useful, what they can do with your project, and how they can use it.
to add a professional README file click here https://github.com/microverseinc/readme-template

Reply:
- [ ] Great job being polite, phrasing your comment as a suggestion and giving a reason for the requested change.